jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -109,8 +110,8 @@ class SpectralExtraction(PluginTemplateMixin, ApertureSubsetSelectMixin,
 
     results_units = Unicode().tag(sync=True)
     spectrum_y_units = Unicode().tag(sync=True)
-    flux_unit = Unicode().tag(sync=True)
-    sb_unit = Unicode().tag(sync=True)
+    flux_units = Unicode().tag(sync=True)
+    sb_units = Unicode().tag(sync=True)
```

Comment:
just to be consistent with the traitlets above, I was confusing myself when writing code later in the plugin and having to switch between singular and plural